src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUsePlatformAnalyzers.cs

```diff
@@ -255,5 +256,52 @@ static void Main()
             buildResult.StdErr.Should().Be(string.Empty);
             buildResult.StdOut.Should().Contain("Program.cs(12,56): warning CA2014: Potential stack overflow. Move the stackalloc out of the loop.");
         }
+
+        [RequiresMSBuildVersionFact("16.8")]
+        public void SDK_imports_the_analyzer_props_file()
+        {
+            var testProject = new TestProject
+            {
+                Name = "HelloWorld",
+                IsSdkProject = true,
+                TargetFrameworks = targetFrameworkNetCore31,
+                IsExe = true,
+                SourceFiles =
+                {
+                    ["Program.cs"] = @"
+                        using System;
+
+                        namespace ConsoleCore
+                        {
+                            class Program
+                            {
+                                static void Main()
+                                {
+                                    while (true)
+                                    {
+                                        Span<char> c = stackalloc char[5];
+                                    }
+                                }
+                            }
+                        }
+                    ",
+                }
+            };
+
+            testProject.AdditionalProperties.Add("CodeAnalysisTreatWarningsAsErrors", "false");
+            testProject.AdditionalProperties.Add("TreatWarningsAsErrors", "true");
+            var testAsset = _testAssetsManager
+                .CreateTestProject(testProject, identifier: "analyzerConsoleApp", targetExtension: ".csproj");
+
+            var buildCommand = new GetValuesCommand(
```

Comment:
A `GetValuesCommand` is for reading a property or item from the build.  It doesn't look like you're doing that here.  Should this be `BuildCommand` instead?